### PR TITLE
Update: Russian translation - Day lengh factor (expansed description and rollback to the old title)

### PR DESCRIPTION
**Pull request by Qwest8K**
I decided that previous title of this setting was more accurate.

Also I expanded its description with that increasing this value will reduce the load on the CPU when calculating "world ticks" (useful on maps larger than 16 777 216 tiles).

### CODE REVIEW

File: src/lang/extra/russian.txt [hunk 1]
```diff
@@ -523,8 +523,8 @@ STR_CONFIG_SETTING_SIMULATE_SIGNALS                             :Симуляц
 STR_CONFIG_SETTING_SIMULATE_SIGNALS_HELPTEXT                    :Устанавливает целевое расстояние между светофорами для новых мостов и туннелей. Фактическое расстояние может немного отличаться от указанного, чтобы избежать неравномерного расположения. Изменение этого параметра не приводит к переоборудованию существующих мостов и туннелей со светофорами.
 STR_CONFIG_SETTING_SIMULATE_SIGNALS_VALUE                       :{COMMA} клет{P ку ки ок}
 
-STR_CONFIG_SETTING_DAY_LENGTH_FACTOR                            :Фактор продолжительности дня: {STRING}
-STR_CONFIG_SETTING_DAY_LENGTH_FACTOR_HELPTEXT                   :Темп игры замедляется из-за этого фактора.
+STR_CONFIG_SETTING_DAY_LENGTH_FACTOR                            :Коэффициент продолжительности дня: {STRING}
+STR_CONFIG_SETTING_DAY_LENGTH_FACTOR_HELPTEXT                   :Темп игры замедляется из-за этого коэффициента. При увеличении снижает нагрузку на процессор путём снижения количества расчётов изменений на карте.
```
Comment (James103) [inline src/lang/extra/russian.txt:527]: Can you please translate the sentence you just added into English, then append your English translation to the setting description in `english.txt`? https://github.com/JGRennison/OpenTTD-patches/blob/e57a7ad2733b74336a3869d0a421841a7046e8ac/src/lang/extra/english.txt#L573

**Comment by Qwest8K** [comment]
@James103 
I hope I translated this correct.

**Comment by JGRennison** [comment]
Performance is not the reason why this setting exists, or a reason why a user should change the setting value, though it can slightly improve performance in some cases.
I don't see that this extra text is really necessary/helpful?

**Comment by Qwest8K** [comment]
Well, in my case, increasing the day length factor has significantly improved performance on maps, which size is exceed limits of original OpenTTD. So I decided to add this note.

**Comment by James103** [comment]
> Well, in my case, increasing the day length factor has significantly improved performance on maps, which size is exceed limits of original OpenTTD. So I decided to add this note.

I can reproduce the same behavior on my end as well.
On a 4k by 4k empty map (16M tiles), the game runs at 13x speed with a day length factor of 1, but at around 50x speed with a day length factor of 4, and at around 80x speed with a day length factor of 120.
On an 8k by 8k empty map (64M tiles), the game runs at 2.4x speed with a day length factor of 1, but at around 9x speed with a day length factor of 4, and at around 13x speed with a day length factor of 120.

A day length factor value as low as 4 is sufficient to reduce the load on the server caused by "World ticks" by around 70% on an empty map (compared to a day length factor of 1), meaning almost all servers will benefit (performance-wise) from increasing the day length factor to a minimum of 4.
Higher day length factors do improve performance by further reducing the load caused by "World ticks," but that performance improvement is marginal at best and should not be viewed as the sole reason to further increase the day length factor.

CPU: Intel Core i5-9400F, 2.90 GHz base clock, 3.90 GHz max boost
OS: Windows 10, 64-bit

**Comment by JGRennison** [comment]
Thanks

**Pull request merged**